pkgs/top-level/all-packages.nix

```diff
@@ -12413,7 +12413,8 @@ let
           ++ lib.optional (cfg.enableTrezor or false) trezor-bridge
          );
       libs = [ gstreamer gst_plugins_base ] ++ lib.optionals (cfg.enableQuakeLive or false)
-             (with xlibs; [ stdenv.cc libX11 libXxf86dga libXxf86vm libXext libXt alsaLib zlib ]);
+             (with xlibs; [ stdenv.cc libX11 libXxf86dga libXxf86vm libXext libXt alsaLib zlib ])
+             ++ lib.optional (enableAdobeFlash && (cfg.enableHalFlash or false)) hal-flash;
```

Comment:
I went with the technical (hal-flash) name for this option, but maybe it would be better if we called this "enableAdobeFlashDRM"?  Let me know what you think.